src/components/MaterialIcon.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react'
+import clsx from 'clsx'
+
+export default function MaterialIcon({
+  className,
+  children,
+  ...props
+}: React.PropsWithChildren<
+  { className?: string } & JSX.IntrinsicElements['i']
+>) {
```

Comment:
```suggestion
}: React.ComponentProps<'i'>) {
```

I think this is what we want.

Reply:
I think you might be right